src/gleam/float.gleam

```diff
@@ -37,7 +37,7 @@ pub fn to_string(f: Float) -> String {
   |> string_builder.to_string
 }
 
-/// Restricts a Float between a lower and upper bound
+/// Restricts a float between a lower and upper bound.
```

Comment:
Could we use uppercase with backticks for int and float in this module? To match the others.

Reply:
Done